manifests/install.pp

```diff
@@ -37,12 +42,11 @@
 
   # anchor, i.e. for config directory set by confd parameter
   file { $conf_dir:
-    ensure  => directory,
+    ensure => directory,
+    mode   => '0755',
```

Comment:
I think this should be 0750 as this is how the (at least Ubuntu) package sets those rights during the installation.